### PR TITLE
Fix city country split

### PR DESCRIPTION
**Pull request by alexey-zakharenkov**
The render uses a text file with lines of the form "City, Country". There occurred a problem with records like "Washington, D.C., USA" which were split incorrectly at the first comma. Assuming that a country name won't contain commas, the pull request rewrites "City, Country" line splitting to happen at the last comma.

### CODE REVIEW

File: render/js/metro.js [hunk 1]
```diff
@@ -66,7 +66,9 @@ L.CitySelect = L.Control.extend({
                 }
 
                 for (var i = 0; i < cities.length; i++) {
-                    city_name = cities[i].split(',')[0];
+                    // Remove country name which follows last comma and doesn't contain commas itself
```
Comment (Zverik) [inline render/js/metro.js]: Why not just `.rsplit(',', 1)[0]`?

Comment (alexey-zakharenkov) [reply render/js/metro.js]: I can't find rfind() in JavaScript. It seems you mixed up JS with Python.

Comment (Zverik) [reply render/js/metro.js]: Oh, it's js. Sorry.

**Comment by alexey-zakharenkov** [comment]
@rostixman please merge the PR using "Squash and merge"

**Pull request merged**